docker-compose.yml

```diff
@@ -51,17 +51,22 @@ services:
 
   clickhouse:
     image: "yandex/clickhouse-server:latest"
-
-  # This is necessary for doing live React dev for the new board, but isn't needed after doing a build
-  # if the nginx config is changed to not redirect /beta/ to port 3000:
+  webserver:
+    image: "nginx:alpine"
+    working_dir: /application
+    volumes:
+      - "./:/application"
+      - "./phpdocker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf"
+    ports:
+      - "${WEBDIP_PORT:-43000}:80"
   beta:
     image: "node:16.15.1-alpine3.14"
     container_name: webdiplomacy-beta
     working_dir: /application
     volumes:
       - "./beta-src:/application"
     ports:
-      - "3000"
+      - "3000:3000" # This is necessary for doing live React dev for the new board, but isn't needed after doing a build. If the nginx config is changed to not redirect /beta/ to port 3000:
```

Comment:
I'm not understanding the second sentence here, is there a clause missing?

Reply:
I just put it back to where it was